src/taco-utils/commands.ts

```diff
@@ -99,11 +99,7 @@ module TacoUtility {
                 // Determine which subcommand we are executing
                 var subCommand = this.getSubCommand(this.data);
                 if (subCommand) {
-                    return subCommand.run(this.data)
-                        .then((telemetryProperties: telemetryHelper.ICommandTelemetryProperties) => {
-                            telemetryProperties["subCommand"] = telemetryHelper.TelemetryHelper.telemetryProperty(subCommand.name, /*isPii*/ false);
```

Comment:
this is removed to handle fallback case. we want to report the actual subcommand instead of "fallback"